src/routes/api/db-migrations/db-migration-utils.ts

```diff
@@ -0,0 +1,101 @@
+import { Request } from "express";
+import fs from "fs";
+import path from "path";
+import { isNodeDev, isNodeTest, isNodeProd, getNodeEnv } from "utils/is-node-env";
+import { getLogger } from "config/logger";
+import { exec as execOrigin } from "child_process";
+import { promisify } from "util";
+
+const exec = promisify(execOrigin);
+const logger = getLogger("DBMigration");
+
+/**
+ *
+ * Functiont to get the parem for db migration.
+ *
+ * Intentially to NOT make this param case insensitive to be more safe
```

Comment:
I'd've removed the double-negation. Maybe something like "Intentially making this param case sensitive for better safety"?